examples/3d/fxaa.rs

```diff
@@ -13,8 +13,8 @@ use bevy::{
 
 fn main() {
     App::new()
-        // Disable MSAA be default
-        .insert_resource(Msaa { samples: 1 })
+        // Disable MSAA by default
+        .insert_resource(Msaa::default())
```

Comment:
```suggestion
        .insert_resource(Msaa::Off)
```
This doesn't match the previous value

Reply:
Woops, thanks